cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -650,7 +651,7 @@ def test_cannot_delete_non_existent_group_configuration(self):
         self.assertEqual(len(user_partititons), 2)
         self.assertEqual(user_partititons[0].name, 'Name 0')
 
-    @ddt.data('content_type_gate', 'enrollment_track')
+    @ddt.data(CONTENT_TYPE_GATING_SCHEME, ENROLLMENT_SCHEME)
```

Comment:
it looks like this is just modifying existing tests, do you think it would make sense to write a test to specifically check for the bug this PR is fixing?

Reply:
Yeah, probably.

Reply:
Ok, I added a test. Check it out?